components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AuthorizationCodeDAOImpl.java

```diff
@@ -110,6 +127,10 @@ public void insertAuthorizationCode(String authzCode, String consumerKey, String
             //insert the hash value of the authorization code
             prepStmt.setString(13, getHashingPersistenceProcessor().getProcessedAuthzCode(authzCode));
             prepStmt.setString(14, getPersistenceProcessor().getProcessedClientId(consumerKey));
+            if (OAuth2ServiceComponentHolder.isIDPIdColumnEnabled()) {
+                prepStmt.setString(15, authenticatedIDP);
```

Comment:
shouldn't this be IDP ID instead of IDP name?

Reply:
This is handled by the query as we don't have the IDP ID at this point.
https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1091/files#diff-595fe8c10d7fa504f02ea9fa378c4921R39

Reply:
in that case since we are already passing tenantID at https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1091/files#diff-8c6619056450d00aabf613330aec77f8L103 , we don't need to pass it again here right?

Reply:
Those are two different params right? will check if there is a way in the query to make use from one input only